source/_static/scss/includes/_base.scss

```diff
@@ -20,16 +20,91 @@ body {
     line-height: $body-line-height;
     color: $body-color;
     background-color: $body-bg;
-    font-weight: $font-wight-normal;
+    font-weight: $font-weight-normal;
     display: flex;
     flex-direction: column;
 }
 
 a {
     color: $link-color;
+    text-decoration: none;
 
     &:hover {
         color: $link-hover-color;
         text-decoration: none;
     }
+}
+
+h1, h2, h3, h4, h5 {
+    color: $docs-item-main-color;
+    & > a.toc-backref {
+        color: $docs-item-main-color;
+    }
+}
+
+div.admonition {
+    & > p.admonition-title {
+        font-weight: bold;
+        font-family: $font-family-base;
+        font-size: 1.2rem;
+    }
+
+    &.warning {
+        background-color: $docs-item-admonition-warning-body;
+        border: none;
+        border-left: 4px solid $docs-item-admonition-warning-border;
+    }
+
+    &.important {
+        background-color: $docs-item-admonition-important-body;
+        border: none;
+        border-left: 4px solid $docs-item-admonition-important-border;
+    }
+
+    &.note {
+        background-color: $docs-item-admonition-note-body;
+        border: none;
+        border-left: 4px solid $docs-item-admonition-note-border;
+    }
+}
+
+dl.minio {
+    margin: 10px 0 10px 0;
+}
+
+div.footer {
```

Comment:
Temporary, the footer is minimally important and this just gets it onto the page.